docs/index.rst

```diff
@@ -1,30 +1,142 @@
-====================================
-Welcome to OpenHatch's documentation
-====================================
+====================
+Welcome to OpenHatch
```

Comment:
The earlier page said "Welcome to OpenHatch's documentation" because I wonder if people, upon arrival, might think that openhatch.readthedocs.org is the real "OpenHatch website". I'm concerned that people might think this is the "site" not the "docs", if we remove that text.

But maybe this is not a serious concern, or maybe it's one that your text changes below address. I'm curious what you think.

I'm -0 to shipping this as-is with that possible source of confusion, but I can live with it.